test/system/src/main/java/io/pravega/test/system/framework/services/PravegaProperties.java

```diff
@@ -0,0 +1,133 @@
+/**
+ * Copyright (c) 2017 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+package io.pravega.test.system.framework.services;
+
+import com.google.common.collect.ImmutableMap;
+import java.util.HashMap;
+import java.util.Map;
+import lombok.Builder;
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * This class encapsulates all system properties to be passed to the operator for a Pravega deployment.
+ * */
+@Slf4j
+@Builder
+public class PravegaProperties {
```

Comment:
Please make the class name more specific and easier to understand. `PravegaProperties` is both vague and ambiguous.

Reply:
I think the name represents exactly what it is.

Reply:
Ok, if you are convinced.